homeassistant/components/wake_on_lan/switch.py

```diff
@@ -32,6 +33,7 @@
         vol.Optional(CONF_HOST): cv.string,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
         vol.Optional(CONF_OFF_ACTION): cv.SCRIPT_SCHEMA,
+        vol.Optional(CONF_UNIQUE_ID): cv.string,
```

Comment:
It already have a perfect unique id in the form of the mac address. Don't add other config option.